sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/CreateSubscriptionOptions.cs

```diff
@@ -297,7 +297,7 @@ public override bool Equals(object obj)
         /// <summary>Determines whether the specified object is equal to the current object.</summary>
         public bool Equals(CreateSubscriptionOptions other)
         {
-            if (other is CreateSubscriptionOptions otherOptions
+            if (other is { } otherOptions
```

Comment:
Any chance you'd be willing to do these others like `CreateQueueOptions`?

Reply:
Absolutely. I just didn't have time yet. Was focused on some Octokit things. Give me 1-2 days more